### PR TITLE
Add volume settings to fart plugin

### PR DESCRIPTION
**Pull request by ItzOnlyAnimal**

### CODE REVIEW

File: src/plugins/fart.ts [hunk 1]
```diff
@@ -22,12 +23,20 @@ export default definePlugin({
...
 
             return {
                 content: (args[0]) ? `<@${args[0].value}> fart` : "fart"
             };
         },
-    }]
+    }],
+    options: {
+        volume: {
+            type: OptionType.NUMBER,
```
Comment (Vendicated) [inline src/plugins/fart.ts:36]: you should add isValid and check if it's 0 < x <= 1.0

Comment (ItzOnlyAnimal) [reply src/plugins/fart.ts:36]: alright will do (alternatively pls add slider @MeguminSama :trollface:)

Comment (Vendicated) [reply src/plugins/fart.ts:36]: slideee

Comment (ItzOnlyAnimal) [reply src/plugins/fart.ts]: oh slider

File: src/plugins/fart.ts [hunk 2]
```diff
@@ -22,12 +23,21 @@ export default definePlugin({
 
         execute(args) {
             const fart = new Audio("https://raw.githubusercontent.com/ItzOnlyAnimal/AliuPlugins/main/fart.mp3");
-            fart.volume = 0.3;
+            fart.volume = Settings.plugins["Fart2"].volume;
```
Comment (Vendicated) [inline src/plugins/fart.ts:26]:
```suggestion
            fart.volume = Settings.plugins.Fart2.volume;
```

Comment (ItzOnlyAnimal) [reply src/plugins/fart.ts:26]: lol this is what happens when u odnt pay attention while kanging

**Pull request merged**